// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.

syntax = "proto3";

package pulumirpc;

// PluginInfo is meta-information about a plugin that is used by the system.
message PluginInfo {
    string version = 1; // the semver for this plugin.
}

// PluginDependency is information about a plugin that a program may depend upon.
message PluginDependency {
    string name = 1;    // the name of the plugin.
    string kind = 2;    // the kind of plugin (e.g., language, etc).
    string version = 3; // the semver for this plugin.
}
